tests/test_parser.py

```diff
@@ -260,3 +260,18 @@ def test_pdf_save_path():
     # Clean up
     os.remove(result["pdf_path"])
     os.rmdir("tests/outputs/temp")
+
+
+@pytest.mark.asyncio
+def test_page_nums():
```

Comment:
Also, add `async` to test `test_page_nums` and `test_pdf_save_path`

Reply:
Fixed